Docs/CommonUI_ViewModel_QuickStart.md

```
@@ -0,0 +1,179 @@
+   - Inherit from `LuaState`
+   - Set `Lua Filename` to your script (e.g., `LuaScripts/ExampleCommonUIWidget.lua`)
+
+2. **Create a Widget Blueprint**
+   - Inherit from `LuaCommonUIWidget`
+   - Set `Lua State` to your LuaState class
+   - Set lifecycle callbacks:
+     - `On Constructed Lua Function`: `"OnWidgetConstructed"`
+     - `On Activated Lua Function`: `"OnWidgetActivated"`
+
+3. **Write Your Lua Script**
+   ```lua
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)